engines/hdb/sound.cpp

```diff
@@ -1688,20 +1678,20 @@ void Sound::playVoice(int index, int actor) {
 	return;
 }
 
-void Sound::setMusicVolume(int volume) {
-	_musicVolume = volume;
-	if (_song1.isPlaying()) {
-		_song1.setVolume(volume);
-	}
-	if (_song2.isPlaying()) {
-		_song2.setVolume(volume);
-	}
-}
+//void Sound::setMusicVolume(int volume) {
+//	_musicVolume = volume;
+//	if (_song1.isPlaying()) {
+//		_song1.setVolume(volume);
+//	}
+//	if (_song2.isPlaying()) {
+//		_song2.setVolume(volume);
+//	}
+//}
```

Comment:
Leftover?

Reply:
Indeed this is leftover, not needed after the changes for sound volume synchronization with ConfMan.